test/test_jit_fuser_legacy.py

```diff
@@ -1,6 +1,7 @@
 import sys
-sys.argv.append("--jit_executor=legacy")
-from test_jit_fuser import *  # noqa: F403
```

Comment:
Hmm, this would make this file a no op instead of rerunning all tests from test_jit_fuser

Reply:
Yeah, I was afraid of doing that. How exactly are these files run normally? I mean what command do I need to invoke so this gets run?